src/Http/TransformsFlexibleErrors.php

```diff
@@ -3,25 +3,32 @@
 namespace Whitecube\NovaFlexibleContent\Http;
 
 use Illuminate\Support\Str;
-use Illuminate\Http\JsonResponse;
-use Symfony\Component\HttpFoundation\Response;
 use Whitecube\NovaFlexibleContent\Flexible;
-use Whitecube\NovaFlexibleContent\Http\FlexibleAttribute;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Response;
 
 trait TransformsFlexibleErrors
 {
     /**
-     * Updates given response's errors for the concerned flexible fields
+     * Checks whether the given response's flexible errors can and should be transformed
      *
-     * @param  \Illuminate\Http\JsonResponse  $response
-     * @return \Illuminate\Http\JsonResponse
+     * @param Response $response
+     * @return bool
      */
-    protected function transformFlexibleErrors(JsonResponse $response)
+    protected function shouldTransformFlexibleErrors(Response $response)
     {
-        if ($response->getStatusCode() === Response::HTTP_UNPROCESSABLE_ENTITY) {
-            $response->setData($this->updateResponseErrors($response->original));
-        }
+        return ($response->getStatusCode() === Response::HTTP_UNPROCESSABLE_ENTITY && get_class($response) === JsonResponse::class);
```

Comment:
Instead of `get_class($response) === JsonResponse::class`, use [`is_a($response, JsonResponse::class)`](https://www.php.net/is_a). Otherwise, seems good to me 👍

Reply:
Committed `get_class` to `is_a`. Also reverted a massive oopsie. I thought `JsonResponse` also extended Illuminate's response. It does not. Thanks!